src/engine/renderer/tr_shade.cpp

```diff
@@ -1587,14 +1579,10 @@ static void Render_forwardLighting_DBS_proj( shaderStage_t *pStage,
 	// bind u_HeightMap
 	if ( pStage->enableReliefMapping )
 	{
-		float depthScale;
-		float reliefDepthScale;
+		float depthScale = RB_EvalExpression( &pStage->depthScaleExp, r_reliefDepthScale->value );
```

Comment:
RB_EvalExpression can have time-dependent values, so there may be reasons not to precompute it.

Reply:
This `RB_EvalExpression()` call do not use precomputed input.